skimage/morphology/tests/test_skeletonize_3d.py

```diff
@@ -126,7 +126,10 @@ def test_skeletonize_num_neighbours():
     circle2 = (ic - 135)**2 + (ir - 150)**2 < 20**2
     image[circle1] = 1
     image[circle2] = 0
-    result = skeletonize_3d(image)
+    with warnings.catch_warnings():
+        # UserWarning for possible precision loss, expected
+        warnings.simplefilter('ignore', UserWarning)
+        result = skeletonize_3d(image)
```

Comment:
Why not use `expected_warnings` here also?

Reply:
You know, I really don't know why.

I guess for that file I copy pasted the warning statement from an other function that ignored warnings that way.


Fixed.